foxtrot-core/src/main/java/com/flipkart/foxtrot/core/reroute/ClusterRerouteConfig.java

```diff
@@ -0,0 +1,20 @@
+package com.flipkart.foxtrot.core.reroute;
+
+import com.flipkart.foxtrot.core.jobs.BaseJobConfig;
+import lombok.Data;
+
+/***
+ Created by mudit.g on Feb, 2019
+ ***/
+@Data
```

Comment:
This job is added to re route shards every day in the morning to balance the cluster